scripts/db_migrator.py

```diff
@@ -485,6 +485,23 @@ def migrate_qos_fieldval_reference_format(self):
         self.migrate_qos_db_fieldval_reference_remove(qos_table_list, self.configDB, self.configDB.CONFIG_DB, '|')
         return True
 
+    def migrate_port_qos_map_global(self):
+        """
+        Generate dscp_to_tc_map for switch.
+        """
+        asics_require_global_dscp_to_tc_map = ["broadcom"]
+        if self.asic_type not in asics_require_global_dscp_to_tc_map:
+            return True
```

Comment:
"return" should do since we are not capturing the return value of this function

Reply:
Fixed. Thanks